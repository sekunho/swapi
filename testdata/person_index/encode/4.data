{"count":82,"next":"https://swapi.dev/api/people/?format=json&page=5","previous":"https://swapi.dev/api/people/?format=json&page=3","results":[{"birth_year":"92.0BBY","created":"2014-12-19T16:54:53.618Z","edited":"2014-12-20T21:17:50.375Z","eye_color":"blue","films":["https://swapi.dev/api/films/4/"],"gender":"male","hair_color":"brown","height":"193","homeworld":"https://swapi.dev/api/planets/28/","mass":"89.0","name":"Qui-Gon Jinn","skin_color":"fair","species":[],"starships":[],"url":"https://swapi.dev/api/people/32/","vehicles":["https://swapi.dev/api/vehicles/38/"]},{"birth_year":"unknown","created":"2014-12-19T17:05:57.357Z","edited":"2014-12-20T21:17:50.377Z","eye_color":"red","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"male","hair_color":"none","height":"191","homeworld":"https://swapi.dev/api/planets/18/","mass":"90.0","name":"Nute Gunray","skin_color":"mottled green","species":["https://swapi.dev/api/species/11/"],"starships":[],"url":"https://swapi.dev/api/people/33/","vehicles":[]},{"birth_year":"91.0BBY","created":"2014-12-19T17:21:45.915Z","edited":"2014-12-20T21:17:50.379Z","eye_color":"blue","films":["https://swapi.dev/api/films/4/"],"gender":"male","hair_color":"blond","height":"170","homeworld":"https://swapi.dev/api/planets/9/","mass":"unknown","name":"Finis Valorum","skin_color":"fair","species":[],"starships":[],"url":"https://swapi.dev/api/people/34/","vehicles":[]},{"birth_year":"46.0BBY","created":"2014-12-19T17:28:26.926Z","edited":"2014-12-20T21:17:50.381Z","eye_color":"brown","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"gender":"female","hair_color":"brown","height":"185","homeworld":"https://swapi.dev/api/planets/8/","mass":"45.0","name":"Padmé Amidala","skin_color":"light","species":[],"starships":["https://swapi.dev/api/starships/39/","https://swapi.dev/api/starships/49/","https://swapi.dev/api/starships/64/"],"url":"https://swapi.dev/api/people/35/","vehicles":[]},{"birth_year":"52.0BBY","created":"2014-12-19T17:29:32.489Z","edited":"2014-12-20T21:17:50.383Z","eye_color":"orange","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/"],"gender":"male","hair_color":"none","height":"196","homeworld":"https://swapi.dev/api/planets/8/","mass":"66.0","name":"Jar Jar Binks","skin_color":"orange","species":["https://swapi.dev/api/species/12/"],"starships":[],"url":"https://swapi.dev/api/people/36/","vehicles":[]},{"birth_year":"unknown","created":"2014-12-19T17:32:56.741Z","edited":"2014-12-20T21:17:50.385Z","eye_color":"orange","films":["https://swapi.dev/api/films/4/"],"gender":"male","hair_color":"none","height":"224","homeworld":"https://swapi.dev/api/planets/8/","mass":"82.0","name":"Roos Tarpals","skin_color":"grey","species":["https://swapi.dev/api/species/12/"],"starships":[],"url":"https://swapi.dev/api/people/37/","vehicles":[]},{"birth_year":"unknown","created":"2014-12-19T17:33:38.909Z","edited":"2014-12-20T21:17:50.388Z","eye_color":"orange","films":["https://swapi.dev/api/films/4/"],"gender":"male","hair_color":"none","height":"206","homeworld":"https://swapi.dev/api/planets/8/","mass":"unknown","name":"Rugor Nass","skin_color":"green","species":["https://swapi.dev/api/species/12/"],"starships":[],"url":"https://swapi.dev/api/people/38/","vehicles":[]},{"birth_year":"unknown","created":"2014-12-19T17:45:01.522Z","edited":"2014-12-20T21:17:50.392Z","eye_color":"blue","films":["https://swapi.dev/api/films/4/"],"gender":"male","hair_color":"brown","height":"183","homeworld":"https://swapi.dev/api/planets/8/","mass":"unknown","name":"Ric Olié","skin_color":"fair","species":[],"starships":["https://swapi.dev/api/starships/40/"],"url":"https://swapi.dev/api/people/39/","vehicles":[]},{"birth_year":"unknown","created":"2014-12-19T17:48:54.647Z","edited":"2014-12-20T21:17:50.395Z","eye_color":"yellow","films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/"],"gender":"male","hair_color":"black","height":"137","homeworld":"https://swapi.dev/api/planets/34/","mass":"unknown","name":"Watto","skin_color":"blue, grey","species":["https://swapi.dev/api/species/13/"],"starships":[],"url":"https://swapi.dev/api/people/40/","vehicles":[]},{"birth_year":"unknown","created":"2014-12-19T17:53:02.586Z","edited":"2014-12-20T21:17:50.397Z","eye_color":"orange","films":["https://swapi.dev/api/films/4/"],"gender":"male","hair_color":"none","height":"112","homeworld":"https://swapi.dev/api/planets/35/","mass":"40.0","name":"Sebulba","skin_color":"grey, red","species":["https://swapi.dev/api/species/14/"],"starships":[],"url":"https://swapi.dev/api/people/41/","vehicles":[]}]}