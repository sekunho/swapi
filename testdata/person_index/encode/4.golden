{"next":"https://swapi.dev/api/people/?format=json&page=5","results":[{"mass":"89.0","height":"193","species":[],"films":["https://swapi.dev/api/films/4/"],"vehicles":["https://swapi.dev/api/vehicles/38/"],"url":"https://swapi.dev/api/people/32/","eye_color":["blue"],"gender":"male","name":"Qui-Gon Jinn","starships":[],"edited":"2014-12-20T21:17:50.375Z","skin_color":"fair","created":"2014-12-19T16:54:53.618Z","hair_color":"brown","birth_year":"92.0BBY","homeworld":"https://swapi.dev/api/planets/28/"},{"mass":"90.0","height":"191","species":["https://swapi.dev/api/species/11/"],"films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/33/","eye_color":["red"],"gender":"male","name":"Nute Gunray","starships":[],"edited":"2014-12-20T21:17:50.377Z","skin_color":"mottled green","created":"2014-12-19T17:05:57.357Z","hair_color":"none","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/18/"},{"mass":"unknown","height":"170","species":[],"films":["https://swapi.dev/api/films/4/"],"vehicles":[],"url":"https://swapi.dev/api/people/34/","eye_color":["blue"],"gender":"male","name":"Finis Valorum","starships":[],"edited":"2014-12-20T21:17:50.379Z","skin_color":"fair","created":"2014-12-19T17:21:45.915Z","hair_color":"blond","birth_year":"91.0BBY","homeworld":"https://swapi.dev/api/planets/9/"},{"mass":"45.0","height":"185","species":[],"films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/","https://swapi.dev/api/films/6/"],"vehicles":[],"url":"https://swapi.dev/api/people/35/","eye_color":["brown"],"gender":"female","name":"Padmé Amidala","starships":["https://swapi.dev/api/starships/39/","https://swapi.dev/api/starships/49/","https://swapi.dev/api/starships/64/"],"edited":"2014-12-20T21:17:50.381Z","skin_color":"light","created":"2014-12-19T17:28:26.926Z","hair_color":"brown","birth_year":"46.0BBY","homeworld":"https://swapi.dev/api/planets/8/"},{"mass":"66.0","height":"196","species":["https://swapi.dev/api/species/12/"],"films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/"],"vehicles":[],"url":"https://swapi.dev/api/people/36/","eye_color":["orange"],"gender":"male","name":"Jar Jar Binks","starships":[],"edited":"2014-12-20T21:17:50.383Z","skin_color":"orange","created":"2014-12-19T17:29:32.489Z","hair_color":"none","birth_year":"52.0BBY","homeworld":"https://swapi.dev/api/planets/8/"},{"mass":"82.0","height":"224","species":["https://swapi.dev/api/species/12/"],"films":["https://swapi.dev/api/films/4/"],"vehicles":[],"url":"https://swapi.dev/api/people/37/","eye_color":["orange"],"gender":"male","name":"Roos Tarpals","starships":[],"edited":"2014-12-20T21:17:50.385Z","skin_color":"grey","created":"2014-12-19T17:32:56.741Z","hair_color":"none","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/8/"},{"mass":"unknown","height":"206","species":["https://swapi.dev/api/species/12/"],"films":["https://swapi.dev/api/films/4/"],"vehicles":[],"url":"https://swapi.dev/api/people/38/","eye_color":["orange"],"gender":"male","name":"Rugor Nass","starships":[],"edited":"2014-12-20T21:17:50.388Z","skin_color":"green","created":"2014-12-19T17:33:38.909Z","hair_color":"none","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/8/"},{"mass":"unknown","height":"183","species":[],"films":["https://swapi.dev/api/films/4/"],"vehicles":[],"url":"https://swapi.dev/api/people/39/","eye_color":["blue"],"gender":"male","name":"Ric Olié","starships":["https://swapi.dev/api/starships/40/"],"edited":"2014-12-20T21:17:50.392Z","skin_color":"fair","created":"2014-12-19T17:45:01.522Z","hair_color":"brown","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/8/"},{"mass":"unknown","height":"137","species":["https://swapi.dev/api/species/13/"],"films":["https://swapi.dev/api/films/4/","https://swapi.dev/api/films/5/"],"vehicles":[],"url":"https://swapi.dev/api/people/40/","eye_color":["yellow"],"gender":"male","name":"Watto","starships":[],"edited":"2014-12-20T21:17:50.395Z","skin_color":"blue, grey","created":"2014-12-19T17:48:54.647Z","hair_color":"black","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/34/"},{"mass":"40.0","height":"112","species":["https://swapi.dev/api/species/14/"],"films":["https://swapi.dev/api/films/4/"],"vehicles":[],"url":"https://swapi.dev/api/people/41/","eye_color":["orange"],"gender":"male","name":"Sebulba","starships":[],"edited":"2014-12-20T21:17:50.397Z","skin_color":"grey, red","created":"2014-12-19T17:53:02.586Z","hair_color":"none","birth_year":"unknown","homeworld":"https://swapi.dev/api/planets/35/"}],"previous":"https://swapi.dev/api/people/?format=json&page=3","count":82}